src/app.js

```diff
@@ -1,7 +1,24 @@
 import { User, Product } from './models';
 import config from './config/config';
+import { DirWatcher } from './dirwatcher';
+import path from 'path';
```

Comment:
I'd recommend you should separate imports of built-in, third-party and your own modules

Reply:
Done